pkg/build/build_test.go

```diff
@@ -343,10 +343,10 @@ func TestSourceDateEpoch(t *testing.T) {
 			want:        time.Unix(1234567890, 0),
 		},
 		{
-			name:            "0",
-			sourceDateEpoch: "0",
+			name:            "Min time",
+			sourceDateEpoch: "315532800",
 			defaultTime:     time.Unix(1234567890, 0),
-			want:            time.Unix(0, 0),
+			want:            time.Unix(315532800, 0),
```

Comment:
Can you add a test for below-min time, which we expect to fail?

Reply:
added.